recipes/nsight-compute/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+
+nsightVersion=$(echo $PKG_VERSION | cut -d. -f1-3)
+nsightLib32="nsight-compute/${nsightVersion}/target/linux-desktop-glibc_2_11_3-x86/"
+
+# Remove 32bit libraries
+[[ -d "${nsightLib32}" ]] && rm -rf ${nsightLib32}
```

Comment:
FWIW this could probably be generalized using something like
```
set +e
rm -rf nsight-compute/*/target/*x86
set -e
```
The set would only be needed if a nonzero exit code is problematic.